CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Bugfix: Fixed the wrong right-click menu showing in the chat input box. (#4177)
 - Bugfix: Fixed popup windows not appearing/minimizing correctly on the Windows taskbar. (#4181)
 - Bugfix: Fixed white border appearing around maximized window on Windows. (#4190)
+- Bugfix: Improved time to open settings (#4193)
```

Comment:
The changelog entry is phrased as if it's in the Minor category rather than the Bugfix category

I'd prefer this to be something along the lines of "Fixed window scaling being applied too many times on startup, causing windows like Settings to be slow."